docs/sources/enterprise/_index.md

```diff
@@ -61,6 +61,7 @@ With Grafana Enterprise, you get access to the following features:
 With a Grafana Enterprise license, you get access to premium data sources, including:
 
 - [AppDynamics](https://grafana.com/plugins/dlopes7-appdynamics-datasource)
+- [Azure Devops](https://grafana.com/grafana/plugins/grafana-azuredevops-datasource/)
```

Comment:
Not relevant to this PR. But can you also make the base URLs of all these to `https://grafana.com/grafana/plugins/` so that they will be consistent. Also let's remove the trailing slash. :) for consistency.

Reply:
I made those changes for the base url and removed trailing slashes for consistency. Thank you!